homeassistant/components/climate/const.py

```diff
@@ -25,6 +25,9 @@ class HVACMode(StrEnum):
     # Device is in Dry/Humidity mode
     DRY = "dry"
 
+    # Device is in Humidify mode
+    HUMIDIFY = "humidify"
```

Comment:
Please don't change the climate entity model.

Humidifying is represented by the humidifier entity in Home Assistant, not a climate entity.